spec.md

```diff
@@ -591,13 +595,41 @@ schema being used.
 }
 ```
 
+##### Deprecation Object 
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| deprecated* | boolean | Indicates that this service or plan is deprecated and will be removed in the future. See [deprecating services and plans](#deprecating-services-and-plans) for more information. |
```

Comment:
there is a redundancy between presence of `deprecation` field in the parent object, and this boolean value.
Another redundancy is the since and eol timestamps.

Reply:
I wanted to allow a broker to say "yo, this was deprecated a year ago, we gave you time."